docs/sources/installation/amazon.md

```diff
@@ -28,8 +28,12 @@ Repository.
    your Amazon Linux instance should be running!
 3. SSH to your instance to install Docker :
    `ssh -i <path to your private key> ec2-user@<your public IP address>`
-4. Once connected to the instance, type
-    `sudo yum install -y docker ; sudo service docker start`
+4. Add the ec2-user to the docker group :
```

Comment:
Thank you for your contribution.  You've contributed at an interesting time. Docker is changing the way it documents installation for cloud vendors.  

It is very difficult to keep up with and test vendor-specific package installation instructions.  This PR is a perfect illustration.  Amazon's own installation instructions *already* included the directive to add the `ec2-user` to the `docker` group.

So, with that philosophy in mind, I'd like to ask you to amend your commit to the following: 

---> https://gist.github.com/moxiegirl/8372356a451ce5c7b93f

page_title: Installation on Amazon EC2
page_description: Installation instructions for Docker on Amazon EC2.
page_keywords: amazon ec2, virtualization, cloud, docker, documentation, installation

## Amazon EC2

You can install Docker on any AWS EC2 Amazon Machine Image (AMI) that runs an
operating system which Docker supports. Amazon's website includes specific
instruction for [installing on Amazon
Linux](https://docs.docker.com/installation/#installation). To install on a
another AMI, follow the installation instructions for its specific operating
system in this installation guide. 

For detailed information on Amazon AWS support for Docker, refer to [Amazon's
documentation
here](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/docker-basics.
htm).